docs/serving/configuration/deployment.md

```diff
@@ -106,3 +109,27 @@ data:
   # List of repositories for which tag to digest resolving should be skipped
   registries-skipping-tag-resolving: registry.example.com
 ```
+
+## Enable Freezer Service
+
+You can configure queue-proxy to call a endpoint when it's traffic drops to zero or scales up from zero by modifying `concurrency-state-endpoint` ConfigMap setting. With this configuration, you can achieve some features like freezing runing process in user-contianer or charing calculation based on time of handling requests。
+
+Before you configure this, your need to implement this endpoint API. The official implementation is [container-freezer](https://github.com/knative-sandbox/container-freezer).
```

Comment:
```suggestion
Before you configure this, you need to implement the endpoint API. The official implementation is [container-freezer](https://github.com/knative-sandbox/container-freezer).
```

Reply:
Does this mean you need to follow the instructions to install the container-freezer daemon in [README](https://github.com/knative-sandbox/container-freezer) for the container-freezer repo?

Reply:
yes, if developer want to use official implementation.

Reply:
fixed